en/00_Getting_Started/05_Recipes.md

```diff
@@ -6,8 +6,6 @@ icon: clipboard
 
 # Adding features to your project with recipes
 
-To achieve more complex use cases in Silverstripe CMS, you may need to combine many modules and add extra configuration to integrate these together. Silverstripe CMS Recipes streamline this process for common use cases.
```

Comment:
Keep this - it's about the concept of recipes and doesn't have anything to do with the specific recipes we're removing support for.

Reply:
Updated